packages/file_selector/file_selector_android/test/file_selector_android_test.dart

```diff
@@ -0,0 +1,83 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:file_selector_android/file_selector_android.dart';
+import 'package:file_selector_platform_interface/file_selector_platform_interface.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+
+  late FileSelectorAndroid plugin;
+  late List<MethodCall> log;
+
+  setUp(() {
+    plugin = FileSelectorAndroid();
+    log = <MethodCall>[];
+    plugin.channel.setMockMethodCallHandler((MethodCall methodCall) async {
```

Comment:
This will need to be changed for Pigeon usage; see file_selector_windows_test.dart for an example of the preferred structure for new tests of Pigeon code.